Library/Homebrew/cask/exceptions.rb

```diff
@@ -136,7 +136,9 @@ def to_s
         Actual:   #{Formatter.error(actual.to_s)}
         File:     #{path}
 
-        To retry an incomplete download, remove the file above.
+        To retry an incomplete download, remove the file above. If the issue persists, visit:
+
```

Comment:
Maybe kill the empty line?

Reply:
I’ve added the line for consistency with our `caveats`. When we add an indented line with a single command / web address, we tend to have the empty line as well.

But if not having the line would make it consistent with other core error messages, I’m fine with it.